subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/TaskErrorExecutionIntegrationTest.groovy

```diff
@@ -15,134 +15,139 @@
  */
 package org.gradle.integtests
 
-import org.gradle.integtests.fixtures.AbstractIntegrationTest
-import org.gradle.integtests.fixtures.executer.ExecutionFailure
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.test.fixtures.file.TestFile
-import org.junit.Test
-
-class TaskErrorExecutionIntegrationTest extends AbstractIntegrationTest {
-    @Test
-    public void reportsTaskActionExecutionFailsWithError() {
-        TestFile buildFile = testFile("build.gradle")
-        buildFile.writelns(
-                "task('do-stuff').doFirst",
-                "{",
-                "throw new ArithmeticException('broken')",
-                "}")
-        ExecutionFailure failure = usingBuildFile(buildFile).withTasks("do-stuff").runWithFailure()
+
+class TaskErrorExecutionIntegrationTest extends AbstractIntegrationSpec {
+    def reportsTaskActionExecutionFailsWithError() {
```

Comment:
⭕️ Do you want to change the test names to Spock style, too?

Reply:
How do you mean?

Reply:
Using spaces instaed of camel case, e.g. “error during task action execution is reported”